Update README (#169)
asetalias/asetalias.github.io: README.md

## ALiAS: Amity Linux Assistance Sapience

>  [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![GitHub license](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 

Reviewer: Why 'GitHub' License?
```suggestion
 [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![License](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 
```

Author: Copied it from the blog README, didn't give it a second look.

![GitHub issues](https://img.shields.io/github/issues-raw/asetalias/asetalias.github.io.svg) ![GitHub repo size](https://img.shields.io/github/repo-size/asetalias/asetalias.github.io.svg)
![Twitter Follow](https://img.shields.io/twitter/follow/asetalias.svg?style=social)

- The following repository contains the source code for the official website of Amity Linux Assistance Sapience. The largest open-source community of Amity University in Noida and now also in Lucknow. 
- The website is hosted on gh-pages itself. 
- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML files.

> <img src="https://user-images.githubusercontent.com/22801822/67155846-3d12c500-f334-11e9-9125-dbf32ea573a9.png" height=480>

Reviewer: I can see the exit full-screen prompt.

Author: Oh, that's embarrassing. Let me add a new one.

## Contribute to the Website 
- Feel free to open an issue or recommend any changes which you want to see on the website. 
- Adhere to the guidelines mentioned below if you want to contribute. 
- Before contributing, be sure to check out the official [wiki](https://github.com/asetalias/asetalias.github.io/wiki)

> ## Contirbute to ALiAS Writes's blog!

Reviewer: ```suggestion
## Contribute to our blog: ALiAS Writes!
```

Author: Good catch!

> ALiAS has its own blog now, [ALiAS Writes](https://asetalias.in/blog). If you like your contributions to be more content, than technical then we we would love to see your blogs be published on ALiAS Writes. [Read more about it here on How to Contibute](https://github.com/asetalias/blog/blob/master/README.md). 

Reviewer: ```suggestion
ALiAS has its own blog now, [ALiAS Writes](https://asetalias.in/blog). If you like your contributions to be more content-centric than technical, we would love to see your blogs on ALiAS Writes. Read more about How to Contribute [here](https://github.com/asetalias/blog/blob/master/README.md). 
```

Reviewer: I feel something like content-centric, content-oriented or content-based is better than just content

- `https://help.github.com/articles/syncing-a-fork/`
- Once you update the forked repo make sure you `rebase` the changes and then open the PR. 
- Ref: `http://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository`
- Upload Screenshot(s) or provide a link to the working demo of the website while making a PR.

## Contributors 

> Thanks to all the generous contributions both offline and online by our members, is behind the magic of everything that ALiAS does. 

Reviewer: I feel like this one doesn't flow well
```suggestion
A big thanks to all the generous contributions both offline and online by our members, who are behind the magic of everything that ALiAS does.
```
